docs/using-capture-growth-charts.md

```diff
@@ -4,58 +4,156 @@
-    next to **Save and add another** and select **Save and exit**.
-
-> **Note**
->
-> Some data elements in an event might be mandatory (marked with a red star next to the data element label).
-> All mandatory data elements must be filled in before the user is allowed to complete the event.
-> The exception to this is if the user has the authority called __"Ignore validation of required fields in Tracker and Event Capture".__
-> If the user has this authority, the mandatory data elements will not be required and
-> the red star will not be displayed next to the data element label. Note that super user that have the __"ALL"__ authority automatically
-> have this authority.
-
-> **Tip**
->
-> The data entry form can also be displayed in **row view**. In this mode the data elements are arranged horizontally. This can be
-> achieved by clicking the **Switch to row view** button on the top right of the data entry form. If you are currently in **row view** you
-> can switch to the default form view by clicking the **Switch to form view** button on the top right of the data entry form.
-
+# Growth chart plugin upload
+Run `yarn build` in the root of the project to build the plugin. The build can be found in `/build/bundle`, and is a compressed file (`.zip`). <br/>
+
+Upload the compressed file to the DHIS2 instance using the `manuall install` funciton in the **App Management** app. Now the plugin should be an available app on the instance and you can find it on this url: 
+- `<Url of instance>/api/apps/capture-growth-chart/plugin.html` 
+
+Remember to alter `<Url of instance>` with the actual url of you instance.
+<br />
+
+# Configuration { #configuration }
+## Maintenance { #maintenance }
+
+### Dataelement
+Create data elements for **Weight**, **Height** and **Head circumference**.
+Weight can be in either `gram` or `kg`, but height and head circumference should be in `cm`.
+ <br />
+
+### Program
+#### Tracked entity attribute
+Tracked entity attribues needed for the Growth chart plugin is `Date of birth` and `gender`. However, `First Name` and `Last Name` are also needed if you want the name to be printed when using the print function. <br /> <br />
+                    
+#### Tracked entity type
+Tracked entity type for person needs the same attributes as the tracked entity attributes. <br /> <br />
+
+
+#### Program     
+##### Attributes
+Select you preffered program for storing the growth variables and displaying the Growth Chart.
+The program should have the following attributes:
+- `First name`
+- `Last name`
+- `Date of birth`
+- `Gender`
+ <br />
+
+##### Program stages
+Select stage where growth variables currently are or will be stored.
+The program stage should have the following data elements:
+- `Weight` (g or kg)
+- `Height` (cm)
+- `Head circumference` (cm)
+<br /> <br />               
+                         
+## Datastore Manangement { #datastore_management }
+### Capture
```

Comment:
I think that this will be a part of the plugin implementation guide and is again outside the scope of this implementation guide for growth charts. In a implementation of this program, the implementation guide for plugins would have to be followed first.